src/Streamly/System/Process.hs

```diff
@@ -11,63 +11,64 @@
 -- used like regular Haskell stream functions connecting them into a stream
 -- pipeline consisting of Haskell functions or other OS processes.
 --
--- The composition is similar to a Posix shell pipeline except that we use @&@
--- instead of @|@. Also note that like the @pipefail@ option in shells,
--- exceptions are propagated if any of the stages fail.
+-- Processes can be composed in a streaming pipeline just like a Posix shell
+-- command pipeline except that we use @&@ instead of @|@. Also note that like
+-- the @pipefail@ option in shells, exceptions are propagated if any of the
+-- stages fail.
 --
+-- The default attributes of the new process created by the APIs in this module
+-- are described below:
+--
+-- * The following attributes are inherited from the parent:
+--
+--     * Current working directory
+--     * Environment
+--     * Process group
+--     * Process uid and gid
+--     * Signal handlers
+--     * Terminal (Session)
+--
+-- * All fds except stdin, stdout and stderr are closed in the child
```

Comment:
Child as in? The right side of `&`?

Reply:
The child system process after forking/execing.